src/types/api.ts

```diff
@@ -158,6 +158,12 @@ export type UserInfo = {
   display_name: string | null;
 };
 
+export type Mention = UserInfo & {
+  userid: string;
+  username: string;
```

Comment:
What is the difference between `userid` and `username` here? Just the format, or can they refer to different users?

Reply:
Yeah, just the format. I guess we could simplify the API to just return the userid and use existing helpers to extract the username from it if needed.

Reply:
Let me clarify, as I was wrong. The `username` is the one the user had at the moment of being mentioned, even if it has changed, while the `userid` will correspond with current value, so we need both.

We discussed yesterday we should probably rename it to `original_username` https://hypothes-is.slack.com/archives/C4K6M7P5E/p1738685990276289?thread_ts=1738685845.418459&cid=C4K6M7P5E